README.md

```diff
@@ -60,10 +60,11 @@ Currently, the following distributions are supported:
 | `adopt-openj9` | Adopt OpenJDK OpenJ9 | [Link](https://adoptopenjdk.net/) | [Link](https://adoptopenjdk.net/about.html) |
 | `liberica` | Liberica JDK | [Link](https://bell-sw.com/) | [Link](https://bell-sw.com/liberica_eula/) |
 | `microsoft` | Microsoft Build of OpenJDK | [Link](https://www.microsoft.com/openjdk) | [Link](https://docs.microsoft.com/java/openjdk/faq)
+| `semeru` | IBM Semeru OpenJDK with OpenJ9 | [Link](https://developer.ibm.com/languages/java/semeru-runtimes/downloads/) | [Link](https://openjdk.java.net/legal/gplv2+ce.html) |
```

Comment:
Done. Are the download and license links OK?

Reply:
You can also use https://ibm.com/semeru-runtimes/downloads which is a little bit shorter but it will expand out to what you have there already, so I don't think it's critical.